tareas/clase-5/tarea-3-clase-5.js

```diff
@@ -6,121 +6,143 @@
 // al apretar el botón "Calcular tiempo total", debe mostrar en un
 // <strong> pre-creado el tiempo total de los videos.
 const $botonCrearInput = document.querySelector(`#agregar`);
+
 $botonCrearInput.onclick = function() {
-    crearInputsTiempo();
+
+    adjuntarHijos('horas');
+    adjuntarHijos('minutos');
+    adjuntarHijos('segundos');
+
 }
 
 const $botonLimpiar = document.querySelector(`#limpiar`);
+
 $botonLimpiar.onclick = function() {
+
     limpiarElementos();
+    borrarResultadosAnteriores();
+
 }
 
 let segundos = 0;
 
 const $botonCalcular = document.querySelector('#calcular');
+
 $botonCalcular.onclick = function() {
+
+    borrarResultadosAnteriores();
     obtenerSegundos();
     calcularMinutosASegundos();
     calcularHorasASegundos();
     mostrarResultados();
-}
 
-function crearInputsTiempo() {
-    crearInputHoras();
-    creatInputsMinutos();
-    crearInputsSegundos();
 }
 
-function crearInputHoras() {
+function crearInputTiempo(tiempo) {
+
     const $nodoPadre = document.querySelector('body');
     const $divHoras = document.createElement('div')
     const $labelHoras = document.createElement("label");
     const $inputHoras = document.createElement("input");
 
-    $nodoPadre.appendChild($divHoras);
-    $divHoras.appendChild($labelHoras);
-    $divHoras.appendChild($inputHoras);
-    $labelHoras.textContent = 'Horas: ';
+    $labelHoras.textContent = `${tiempo[0].toUpperCase()+tiempo.substring(1)}: `;
     $inputHoras.type = 'number';
-    $inputHoras.className = 'horas';
-}
+    $inputHoras.className = `${tiempo}`;
 
-function creatInputsMinutos() {
-    const $nodoPadre = document.querySelector("body");
-    const $divMinutos = document.createElement('div');
-    const $labelMinutos = document.createElement("label");
-    const $inputMinutos = document.createElement("input");
-    
-    $nodoPadre.appendChild($divMinutos);
-    $divMinutos.appendChild($labelMinutos);
-    $divMinutos.appendChild($inputMinutos);
-    $labelMinutos.textContent = 'Minutos: ';
-    $inputMinutos.type = 'number';
-    $inputMinutos.className = 'minutos';
+    return {$nodoPadre,$divHoras,$labelHoras,$inputHoras}
 }
 
-function crearInputsSegundos() {
-    const $nodoPadre = document.querySelector("body");
-    const $divSegundos = document.createElement('div');
-    const $labelSegundos = document.createElement("label");
-    const $inputSegundos = document.createElement("input");
-    
-    $nodoPadre.appendChild($divSegundos);
-    $divSegundos.appendChild($labelSegundos);
-    $divSegundos.appendChild($inputSegundos);
-    $labelSegundos.textContent = 'Segundos: ';
-    $inputSegundos.type = 'number';
-    $inputSegundos.className = 'segundos';
+function adjuntarHijos(tiempo) {
+
+    const {$nodoPadre,$divHoras,$labelHoras,$inputHoras} = crearInputTiempo(`${tiempo}`);
```

Comment:
estás usando template literals para algo que no está siendo interpolado

```suggestion
    const {$nodoPadre,$divHoras,$labelHoras,$inputHoras} = crearInputTiempo(tiempo);
```

Reply:
Bueno, espero que este bien ahora jajaj, https://github.com/Kund097/introduccion-a-js/pull/8/

Reply:
y muchas gracias por tomarte el tiempo de ver y marcar los errores! me sirve muchisimo, que pases lindas vacaciones, nada que ver por aca no?